setup.py

```diff
@@ -7,8 +7,8 @@
     version='0.2',
     description='High-level file IO library',
     url='https://github.com/CAIDA/pywandio',
-    author='Alistair King, Chiara Orsini',
-    author_email='bgpstream-info@caida.org',
+    author='Alistair King, Mingwei Zhang',
```

Comment:
why did you remove chiara's name from the author list?

Reply:
Sorry, my mistake! I only looked at git commit authors. Added her back in f255036